components/datetime/src/provider/calendar/symbols.rs

```diff
@@ -56,24 +59,28 @@ macro_rules! symbols {
         ($name: ident { $element: ident: Option<$ty: ty>, $($tokens: tt)+ } -> ($($members:tt)*)) => {
             symbols!($name { $($tokens)* } -> (
                 $($members)*
+                #[cfg_attr(feature = "provider_serde", serde(borrow))]
                 pub $element: Option<$ty>,
```

Comment:
Issue: serde(borrow) does not work through options.

Reply:
Please add tests to verify that the data being deserialized is actually being borrowed.  `#[serde(borrow)]` is weird and it sometimes works and sometimes doesn't work.

Reply:
Ugh i was looking for `Option` but didn't notice this one since it's in a macro

Reply:
We can probably get a clippy lint that warns about serde(borrow) on option fields